db/re/skill_db.yml

```
@@ -43267,6 +43267,75 @@ Body:
         - Level: 5
           Amount: 82
       State: Mado
+  - Id: 6509
+    Name: BO_MYSTERY_POWDER
+    Description: Mystery Powder
+    MaxLevel: 5
+    Type: Weapon
+    TargetType: Attack
+    DamageFlags:
+      Splash: true
+    Range: 3
+    Hit: Multi_Hit
+    HitCount: -5
+    Element: Weapon
+    SplashArea:
+      - Level: 1
+        Area: 2
+      - Level: 2
+        Area: 2
+      - Level: 3
+        Area: 2
+      - Level: 4
+        Area: 3
+      - Level: 5
+        Area: 3
+    Duration1: 60000
+    AfterCastActDelay: 500
+    Cooldown: 1500
+    Requires:
+      SpCost: 82
+    Status: Mystery_Powder
+  - Id: 6510
+    Name: BO_DUST_EXPLOSION
+    Description: Dust Explosion
+    MaxLevel: 5
+    Type: Weapon
+    TargetType: Attack
+    DamageFlags:
+      Splash: true
+    Range: 3
+    Hit: Multi_Hit
+    HitCount: -5
```

```suggestion
    HitCount: 5
```
Damage is increased by hits